lib/publish_organisations_homepage.rb

```diff
@@ -0,0 +1,105 @@
+class PublishOrganisationsHomepage
+  def publish
+    payload = present_for_publishing_api
+    Services.publishing_api.put_content(payload[:content_id], payload[:content])
+    Services.publishing_api.publish(payload[:content_id], nil, locale: "en")
+  end
+
+private
+
+  def base_path
```

Comment:
`base_path` and `content_id` can be constants perhaps

Reply:
Yup, done